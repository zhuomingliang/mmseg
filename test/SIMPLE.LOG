ASCII characters = 129
BIG-5 symbols = 0
BIG-5 Chinese characters = 148
BIG-5 Chinese words = 84
Average BIG-5 Chinese word length = 1.761905
