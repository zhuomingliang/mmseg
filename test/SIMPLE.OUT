Some "tricky" sentences:

The following sentences are from Yeh & Lee (1991)
  研究生 命 起源 
  颱風眼 看 就要 來了 
  這個 記者會 說 國語 
  幾個 戴 眼鏡 的 男人 在 其中 一人 家中 聚會 
  鄰家 的 小女 孩子 們 
  人為萬物之靈 
  解救 國家 
  如果 一個人 生 三 子 便 得獎 
  想法 國 的 民族 和 世界上 的 民族 
  用 各國 人口 的 增加 數 
  來到 了 這 座 遠近 聞名 的 道觀 
  這 主要 是 因為 他 所能 提供 的 高 解析度 及 低 噪音 

The following sentences are from Fan & Tsai (1988)
  已經 過了 二十 年 
  大學生 活 多采多姿 
  把 他的 確實 行動 作了 分析 
